manifests/w/Wondershare/Filmora/10.5.yaml

```diff
@@ -0,0 +1,23 @@
+PackageIdentifier: "Wondershare.Filmora"
+PackageVersion: "10.5"
+PackageLocale: "en-US"
+Publisher: "Wondershare Software Co. Ltd."
+PublisherUrl: "https://filmora.wondershare.com/"
+PrivacyUrl: "https://www.wondershare.com/privacy.html"
+PackageName: "Wondershare Filmora"
+License: "EULA"
+ShortDescription: "Intuitive video editing"
+Tags: 
+ - "Video Editor"
+ - "Wondershare"
+ - "Filmora"
+InstallerType: "exe"
+InstallModes: 
+ - "interactive"
+Installers: 
+ - Architecture: "x64"
+   InstallerType: "exe"
+   InstallerUrl: "https://download.wondershare.com/inst/filmora_setup_full846.exe"
+   InstallerSha256: SHA256: AFCCF501B6D3CA107F5B83CF73B4735E90DB2AD79C490F1DD4F790342EDBB98B
```

Comment:
`InstallerSha256: SHA256: AFCCF501B6D3CA107F5B83CF73B4735E90DB2AD79C490F1DD4F790342EDBB98B`
This was the issue with this PR btw.